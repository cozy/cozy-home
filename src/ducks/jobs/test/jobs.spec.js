/* eslint-env jest */

import { isJobRunning } from '../'

describe('Jobs Duck', () => {
  describe('isJobRunning', () => {
    ;['queued', 'running'].forEach(state => {
      it(`returns true for ${state} job`, () => {
        expect(isJobRunning({}, { state: state })).toMatchSnapshot()
      })
    })
    ;[('done', 'error')].forEach(state => {
      it(`returns true for ${state} job`, () => {
        expect(isJobRunning({}, { state: state })).toMatchSnapshot()
      })
    })
  })
})
