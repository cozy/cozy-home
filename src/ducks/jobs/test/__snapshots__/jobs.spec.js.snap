// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jobs Duck isJobRunning returns true for error job 1`] = `false`;

exports[`Jobs Duck isJobRunning returns true for queued job 1`] = `true`;

exports[`Jobs Duck isJobRunning returns true for running job 1`] = `true`;
